common/base/kustomization.yaml

```
@@ -3,3 +3,4 @@ kind: Kustomization
 resources:
   - ../helm-repositories/
   - ../values/
+  - ../operator/
```

Just to be consistent with the chart:
```suggestion
  - ../kommander-operator/
```

---

[30ce6f8](https://github.com/mesosphere/kommander-applications/pull/937/commits/30ce6f80fa9229a4f38655da0a77b24562390e93)